projects/soft_999_eddy_kim_fw_102/src/main.c

```diff
@@ -0,0 +1,44 @@
+#include <stdint.h>
+#include <stdio.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define COUNTER_A_WAIT_TIME 500
```

Comment:
Awesome that you factored this out into a constant! Our convention is to suffix constants like this with their unit - in this case it would be `_MS`.